Automattic/wp-calypso
# Marketing: Fix Traffic Page for Jetpack Non-Admins

#### Changes proposed in this Pull Request

* This fixes a defect of #34597 (oops!) that I just noticed now, where this is appearing for non-admins on a Jetpack-connected site. An error is triggered if a non-admin tries to toggle a setting.

<img width="1262" alt="Screenshot 2019-09-18 at 16 54 00" src="https://user-images.githubusercontent.com/43215253/65164981-d26d2180-da35-11e9-9768-57ac44fd9066.png">

#### Testing instructions

Visit `/marketing/traffic/site` on a Jetpack-connected site as a non-admin. Verify that only the Empty Content is rendered. 

 cc @donpark
